traffic_ops/app/db/migrations/2021112117412459_logs_change.up.sql

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with this
+ * work for additional information regarding copyright ownership.  The ASF
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+CREATE TABLE public.log_new (
+	"message" text NOT NULL,
+	"user" bigint NOT NULL,
+	"date" timestamp with time zone NOT NULL DEFAULT NOW()
```

Comment:
`last_updated` is kind of ubiquitous in our API, even though it doesn't necessarily make sense for non-updatable things. Making it unique in this sense means we can't easily reuse queries for things like IMS using the ubiquitous `last_updated` column.

Reply:
Well I could always leave the column name alone but just change the struct property name. You think that's okay?

Reply:
That might be confusing

Reply:
I think it'd be fine, especially if I added `db` struct tags to make the relationship explicit. But if you don't agree I'll just make it `lastUpdated` the whole way through and make note in the documentation that the name was chosen for consistency rather than clarity.

Reply:
IMO I think the consistency across the API is more useful here than the clarity of this particular endpoint